airstack-modules/modules/airstack/common/index.ts

```diff
@@ -44,13 +48,44 @@ AIR_NETWORK_MAP.set("osmosis", "OSMOSIS");
 AIR_NETWORK_MAP.set("matic", "MATIC");
 AIR_NETWORK_MAP.set("xdai", "XDAI");
 
-export function processNetwork(network: string): string {
+const AIR_CHAIN_ID_MAP = new TypedMap<string, string>();
+AIR_CHAIN_ID_MAP.set("arbitrum-one", "42161");
+AIR_CHAIN_ID_MAP.set("arweave-mainnet", "174");
+AIR_CHAIN_ID_MAP.set("aurora", "1313161554");
+AIR_CHAIN_ID_MAP.set("avalanche", "43114");
+AIR_CHAIN_ID_MAP.set("boba", "288");
+AIR_CHAIN_ID_MAP.set("bsc", "56");
+AIR_CHAIN_ID_MAP.set("celo", "42220");
+AIR_CHAIN_ID_MAP.set("COSMOS", "cosmos");
+AIR_CHAIN_ID_MAP.set("CRONOS", "25");
+AIR_CHAIN_ID_MAP.set("mainnet", "1");
+AIR_CHAIN_ID_MAP.set("goerli", "5");
+AIR_CHAIN_ID_MAP.set("fantom", "250");
+AIR_CHAIN_ID_MAP.set("fuse", "122");
+AIR_CHAIN_ID_MAP.set("harmony", "1666600000");
+AIR_CHAIN_ID_MAP.set("juno", "juno-1");
+AIR_CHAIN_ID_MAP.set("moonbeam", "1284");
+AIR_CHAIN_ID_MAP.set("moonriver", "1285");
+AIR_CHAIN_ID_MAP.set("near-mainnet", "1313161554");
+AIR_CHAIN_ID_MAP.set("optimism", "10");
+AIR_CHAIN_ID_MAP.set("osmosis", "osmosis-1");
+AIR_CHAIN_ID_MAP.set("matic", "137");
+AIR_CHAIN_ID_MAP.set("xdai", "100");
+
+export function getNetwork(network: string): string {
   const value = AIR_NETWORK_MAP.get(network);
   const result: string = value !== null ? value : "unknown";
   return result;
 }
 
-//air entity funcitons
+export function getChainId(): string {
+  let network = dataSource.network();
+  const value = AIR_CHAIN_ID_MAP.get(network);
+  const result: string = value !== null ? value : "unknown";
```

Comment:
Instead of returning `unknown` it should throw an error. Similarly for `getNetwork`

Reply:
throwing error now